test/node.test.js

```diff
@@ -39,3 +39,146 @@ it('supports current node version', () => {
   expect(versions).toHaveLength(1)
   expect(versions[0]).toMatch(/node \d+\.\d+\.\d+/)
 })
+
+it('supports comparison operator', () => {
+  expect(browserslist('node <= 5')).toEqual([
+    'node 5.0.0',
+
+    'node 4.9.0',
+    'node 4.8.0',
+    'node 4.7.0',
+    'node 4.6.0',
+    'node 4.5.0',
+    'node 4.4.0',
+    'node 4.3.0',
+    'node 4.2.0',
+    'node 4.1.0',
+    'node 4.0.0',
+
+    'node 0.9.0',
+    'node 0.8.0',
+    'node 0.7.0',
+    'node 0.6.0',
+    'node 0.5.0',
+    'node 0.4.0',
+    'node 0.3.0',
+    'node 0.2.0',
+    'node 0.12.0',
+    'node 0.11.0',
+    'node 0.10.0'
+  ])
+
+  expect(browserslist('node < 5')).toEqual([
+    'node 4.9.0',
+    'node 4.8.0',
+    'node 4.7.0',
+    'node 4.6.0',
+    'node 4.5.0',
+    'node 4.4.0',
+    'node 4.3.0',
+    'node 4.2.0',
+    'node 4.1.0',
+    'node 4.0.0',
+
+    'node 0.9.0',
+    'node 0.8.0',
+    'node 0.7.0',
+    'node 0.6.0',
+    'node 0.5.0',
+    'node 0.4.0',
+    'node 0.3.0',
+    'node 0.2.0',
+    'node 0.12.0',
+    'node 0.11.0',
+    'node 0.10.0'
+  ])
+
+  expect(browserslist('node >= 9')).toEqual(expect.arrayContaining([
```

Comment:
Note, greater than test use `expect.arrayContaining` to avoid breaking in the future once Node is updated